tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -299,7 +301,7 @@ Retrieving a single article is very simple:
 pub fn (app &App) retrieve_article() ?Article {
 	return sql app.db {
 		select from Article limit 1
-	}
+	} or { panic(err) }
```

Comment:
`}[0] or { panic(err) }` may fix the compilation error

Reply:
now this part is failing 
```v
article := app.retrieve_article(10) or {
	app.text('Article not found')
	return
}
```
because its passing an argument (10), not sure exactly what the expected sql to be invoked

Reply:
Should be perhaps: `return app.text('Article not found')`

Reply:
ok updated , and I just removed the argument